src/goTest.ts

```diff
@@ -112,12 +116,32 @@ export function testCurrentFile(timeout: string) {
 }
 
 /**
+ * Runs the previously executed test.
+ */
+export function testPrevious() {
+	let editor = vscode.window.activeTextEditor;
+	if (!editor) {
```

Comment:
Why does an editor need to be active in this case?  It sounds like the semantics for this command are not tied to what is in the current editor (if there is one).  It just executes whatever the last Go test was that was invoked.